linera-views/src/journaling.rs

```diff
@@ -87,8 +96,8 @@ struct JournalHeader {
 /// for bypassing the limits on the batch size
 #[derive(Clone)]
 pub struct JournalingKeyValueStore<K> {
-    /// The inner client that is called by the LRU cache one
-    pub client: K,
+    /// The inner store that is called by the LRU cache one
```

Comment:
```suggestion
    /// The inner store that is called by the journaling one
```

Or something like: "The wrapped store on top of which journaling is implemented"